_posts/2020-11-19-Setting-Up-a-Pull-Request.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Setting Up a Pull Request
+description: 
+summary: 
+tags: 
+---
+Kajari instructed Celeste and I to start creating a new post using pull request. Pull requests are changes committed in the branch that are prresented to viewers of the users repository on Github. To initiate a new post/pull request, the following commands must be used to complete the task.
+* git checkout -b new post - the purpose of checkout -b is to create a new branch with the name new_post
```

Comment:
Also explain that the branch can have any name. It should usually be a reference to the content of the new post. And can contain information such as who created it. For instance it can be `osvaldo/pull-request-post`.